perfkitbenchmarker/beam_benchmark_helper.py

```diff
@@ -192,44 +217,34 @@ def _BuildMavenCommand(benchmark_spec, classname, job_arguments):
     cmd: Array containing the built command.
   """
   cmd = []
-  maven_executable = FLAGS.maven_binary
 
-  if not vm_util.ExecutableOnPath(maven_executable):
+  gradle_executable = _GetGradleCommand()
+
+  if not vm_util.ExecutableOnPath(gradle_executable):
     raise errors.Setup.MissingExecutableError(
-        'Could not find required executable "%s"' % maven_executable)
-  cmd.append(maven_executable)
+      'Could not find required executable "%s"' % gradle_executable)
 
-  cmd.append('-e')
-  cmd.append('verify')
-  cmd.append('-Dit.test={}'.format(classname))
-  cmd.append('-DskipITs=false')
+  cmd.append(gradle_executable)
+  cmd.append('integrationTest')
+  cmd.append('--tests={}'.format(classname))
 
   if FLAGS.beam_it_module:
-    cmd.append('-pl')
+    cmd.append('-p')
     cmd.append(FLAGS.beam_it_module)
 
-  if FLAGS.beam_it_profile:
-    cmd.append('-P{}'.format(FLAGS.beam_it_profile))
-
   beam_args = job_arguments if job_arguments else []
 
-  # Don't add any args when the user overrides beam_runner_profile since it is
-  # expected that they know what they are doing and we can't know what args
-  # to pass since it differs by runner.
-  if (benchmark_spec.service_type == dpb_service.DATAFLOW
-      and (FLAGS.beam_runner_profile is not None and
-           len(FLAGS.beam_runner_profile) > 0)):
-    beam_args.append('"--defaultWorkerLogLevel={}"'.format(FLAGS.dpb_log_level))
-
-  AddRunnerProfileMvnArgument(benchmark_spec.service_type, cmd,
-                              FLAGS.beam_runner_profile)
-  AddRunnerOptionMvnArgument(benchmark_spec.service_type, beam_args,
-                             FLAGS.beam_runner_option)
-  AddExtraMvnProperties(cmd, FLAGS.beam_extra_mvn_properties)
+  AddRunnerArgument(cmd)
+  AddRunnerPipelineOption(beam_args)
+  AddFilesystemArgument(cmd)
+  AddExtraProperties(cmd)
 
   cmd.append("-DintegrationTestPipelineOptions="
```

Comment:
be consistent about quotes. Stick with ' ?

Reply:
ok